lib/downloadManager.js

```diff
@@ -157,28 +167,35 @@ async function unzipPkg(zipFileDir, unzipDir, filterStr) {
 
 async function downloadSamsungCertfile() {
     console.info('[webide-common-tizentv]downloadSamsungCertfile start');
-	  let certGenFoler = 'https://developer.samsung.com/sdk-manager/repository/tizen-certificate-extension_2.0.42.zip';
-	  if (!fs.existsSync(tmpDir)) {
-		    console.info('[webide-common-tizentv]create dir:' + tmpDir);
-		    fs.mkdirSync(tmpDir);
-	  }
-
-	  let streamFile = fs.createWriteStream(certZipFile);
-	  console.info('[webide-common-tizentv]requet to download ' + certGenFoler);
-	  await request(certGenFoler).pipe(streamFile).on('close',async () => {
-		    console.info('[webide-common-tizentv]downloadSamsungCertfile(): download certificate file successful');
-		    //unzipSamsungCertifile();
+    let certGenFoler = 'https://developer.samsung.com/sdk-manager/repository/tizen-certificate-extension_2.0.42.zip';
```

Comment:
Refer to your code, the `tizen-certificate-extesion` is disable to change version. Can you get a latest version of the `tizen-certificate-extension`?

Reply:
And one more thing, what is a 'foler'? Is it a typo?

Reply:
done